content/en/docs/tasks/access-application-cluster/list-all-running-container-images.md

```diff
@@ -51,6 +51,18 @@ the `.items[*]` portion of the path should be omitted because a single
 Pod is returned instead of a list of items.
 {{< /note >}}
 
+## List all images of the workload resources
+
+Listing Pods may yield different results based on the Pods defined lifecycle.
+
+The following example lists all the images possible run in the cluster, except 
+for Orphaned Pods created directly without using workload resources. 
+
+```shell
+kubectl get Deployment,ReplicaSet,StatefulSet,DaemonSet,Job,CronJob --all-namespaces \
+        -o=custom-columns='NAME:metadata.namespace,KIND:kind,NAME:metadata.name,IMAGE:..image'
```

Comment:
What if the workload has more than one container?

Reply:
Here is an example for such case:

```
NAMESPACE   KIND          NAME                                           IMAGE
devops      StatefulSet   devops-database-mysql-cluster-mysql            quay.io/presslabs/mysql-operator-sidecar:0.4.0,percona@sha256:713c1817615b333b17d0fbd252b0ccc53c48a665d4cfcb42178167435a957322,percona@sha256:713c1817615b333b17d0fbd252b0ccc53c48a665d4cfcb42178167435a957322,quay.io/presslabs/mysql-operator-sidecar:0.4.0,prom/mysqld-exporter:master,quay.io/presslabs/mysql-operator-sidecar:0.4.0,quay.io/presslabs/mysql-operator-sidecar:0.4.0
```